### PR TITLE
Replace JAVA_VERSION with JDK_VERSION

### PR DESCRIPTION
**Pull request by renfeiw**
- Update all subset entries in playlist.xml
- Replace JAVA_VERION with JDK_VERSION in build.xml
- Update mkgen.pl and testKitGen.pl
- Update readme
- Rename file names that contain SE version

[ci skip]

Signed-off-by: Renfei Wang <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
Reviewers: @llxia @smlambert

File: test/TestConfig/scripts/build_test.xml [hunk 1]
```diff
@@ -67,11 +67,11 @@
 	</if>
 	<echo> build.list is ${build.list} </echo>
 
-	<target name="build" depends="stage_test_material" description="using ${JAVA_VERSION} compile the source ">
+	<target name="build" depends="stage_test_material" description="using ${JDK_VERSION} compile the source ">
```
Comment (llxia) [inline test/TestConfig/scripts/build_test.xml]: Using java ${JDK_VERSION} to compile the source

File: test/docs/OpenJ9TestUserGuide.md [hunk 2]
```diff
@@ -47,8 +47,8 @@ tools should be installed on your test machine to run tests.
     ```
     JAVA_BIN=<path to JDK bin directory that you wish to test>
     SPEC=[linux_x86-64|linux_x86-64_cmprssptrs|...] (platform on which to test)
-    JAVA_VERSION=[SE80|SE90|SE100|SE110|Panama|Valhalla] (SE90 default value)
-    JAVA_IMPL=[openj9|ibm|hotspot|sap] (openj9 default value)
+    JDK_VERSION=[8|9|10|11|Panama|Valhalla] (9 default value)
```
Comment (llxia) [inline test/docs/OpenJ9TestUserGuide.md]: maybe we should change default to 8. 9 is not supported any more.

File: test/TestConfig/scripts/build_test.xml (hunk 1 above)
Comment (renfeiw) [reply test/TestConfig/scripts/build_test.xml]: updated

File: test/docs/OpenJ9TestUserGuide.md (hunk 2 above)
Comment (renfeiw) [reply test/docs/OpenJ9TestUserGuide.md]: updated

**Comment by llxia** [comment]
Jenkins test sanity all all

**Comment by llxia** [comment]
jenkins test sanity xlinux,zlinux,plinux,aix,win32,win all

**Comment by llxia** [comment]
jenkins test sanity xlinux,zlinux,plinux,aix jdk8

**Comment by llxia** [comment]
jenkins test sanity xlinux,zlinux,plinux,aix jdk10

**Comment by smlambert** [comment]
General question, for the playlist entries where 9-11 are defined, is it a good time to just remove the subset/subsets tags altogether, so that targets for all version will be generated?  (I recognize at some point and possibly still we accept Panama, Valhalla, Amber as 'subsets').

**Comment by renfeiw** [comment]
I will remove subsets for 8-11 in the next commit with the change to remove Panama and Valhalla.

**Comment by llxia** [comment]
Jenkins test sanity xlinux jdk11

**Comment by llxia** [comment]
jenkins test sanity win jdk8

**Comment by llxia** [comment]
jenkins test sanity plinux jdk10

**Comment by llxia** [comment]
jenkins test sanity aix jdk11

**Comment by llxia** [comment]
jenkins test sanity plinux jdk10

**Comment by llxia** [comment]
jenkins test sanity win jdk8

**Comment by llxia** [comment]
jenkins test sanity plinux jdk10

**Comment by llxia** [comment]
jenkins test sanity aix jdk11

**Comment by llxia** [comment]
jenkins test sanity win jdk8

**Comment by llxia** [comment]
jenkins test sanity plinux jdk10

**Comment by llxia** [comment]
jenkins test sanity aix jdk11

**Comment by llxia** [comment]
aix failed with a known issue #2801

**Comment by renfeiw** [comment]
@sophia-guo yes, a followup PR is https://github.com/eclipse/openj9/pull/2930

**Comment by llxia** [comment]
Currently we support both JAVA_VERSION and JDK_VERSION. JAVA_VERSION will be removed eventually. 
fyi @AdamBrousseau @jdekonin @vsebe

**Pull request merged**